nequip/data/AtomicData.py

```diff
@@ -500,6 +528,22 @@ def to_ase(self, type_mapper=None) -> Union[List[ase.Atoms], ase.Atoms]:
                     fields["forces"] = force[mask].cpu().numpy()
                 mol.calc = SinglePointCalculator(mol, **fields)
 
+            # add other information
+            for key in extra_fields:
+                if key in _NODE_FIELDS:
+                    # mask it
+                    mol.arrays[key] = self[key][mask].cpu().numpy()
+                elif key in _EDGE_FIELDS:
+                    mol.info[key] = self[key][edge_mask].cpu().numpy()
+                elif key == AtomicDataDict.EDGE_INDEX_KEY:
+                    mol.info[key] = self[key][:, edge_mask].cpu().numpy()
+                elif key in _GRAPH_FIELDS:
```

Comment:
question. The register step of the graph field may not happen in the nequip-evaluate. Is this tested?